easybuild/easyblocks/f/fdtd_solutions.py

```diff
@@ -40,7 +44,23 @@ def configure_step(self):
 
     def build_step(self):
         """No build step for FDTD Solutions."""
-        pass
+        # locate RPM and rebuild it to make it relocatable
+        rpms = glob.glob(os.path.join(self.cfg['start_dir'], 'rpm_install_files', 'FDTD-%s*.rpm' % self.version))
```

Comment:
I just don't want to make any assumptions here. In the versions I've tested have 'el5' in them, but I don't think that's a safe assumption to make for future versions.